common/schemes/wc_stasis_trap_scheme.txt

```diff
@@ -0,0 +1,269 @@
+
+	power_per_skill_point = 2.0
+	resistance_per_skill_point = 1.75
+	power_per_agent_skill_point = 1
+	
+	minimum_progress_chance = 10
+	maximum_progress_chance = 90
+	minimum_success = 5
+	maximum_secrecy = 85
+	maximum_success = 85
+	
+	allow = {
+		has_perk = stasis_trap # Scheme unlocked by the Stasis Trap
+		
+		is_adult = yes
+		is_landed = yes
+		is_imprisoned = no
+		is_in_stun_trigger = no
+	}
+	
+	valid = {
+		is_landed = yes
+		OR = {
+			NOT = { is_parent_of = scope:target }
+			has_trait_with_flag = can_murder_own_children
+		}
+		scope:target = {
+			is_in_stun_trigger = no
+			exists = location
+			in_diplomatic_range = scope:owner 
+		}
+		# Special AI exceptions
+		NOT = {
+			scope:owner = {
+				is_ai = yes
+				has_opinion_modifier = {
+					target = scope:target
+					modifier = repentant_opinion
+				}
+			}
+		}
+	}
+
+	valid_agent = {
+		age >= 14
+	}
+	
+	agent_join_chance = {
+		base = 0
+		
+		# Base Reluctance, including negative AI Energy
+		hostile_scheme_base_reluctance = yes
+
+		#Is Owner sending a gift in the invite interaction?
+		hostile_scheme_with_gift_modifier = yes
+
+		#Default Hostile Scheme modifiers
+		hostile_murder_agent_base_join_chance_modifier = yes
+	}
+	
+	base_success_chance = {
+		base = 0
+		
+		hostile_scheme_base_chance_modifier = yes
+		
+		# Spiritual Head of Faith
+		modifier = {
+			add = -50
+			desc = "SCHEME_VS_SPIRITUAL_HOF"
+			scope:target = {
+				faith = scope:owner.faith
+				faith = {
+					has_doctrine = doctrine_spiritual_head
+				}
+				faith = {
+					exists = religious_head
+					religious_head = {
+			 			this = scope:target
+			 		}
+				}
+			}
+		}
+		
+		# Various penalties
+		first_valid = {
+			modifier = { # Foreign Rulers are very hard to stun
+				add = -75
+				desc = "sway_foreign_target"
+				scope:target = {
+					is_ruler = yes
+					NOT = {
+						scope:target.top_liege = scope:owner.top_liege
+					}
+				}
+			}
+			modifier = { # Foreign subjects less so
+				add = -40
+				desc = "sway_foreign_target"
+				scope:target = {
+					is_foreign_court_or_pool_guest = no
+					NOT = {
+						scope:target.top_liege = scope:owner.top_liege
+					}
+				}
+			}
+			modifier = { # Rulers are harder to stun
+				add = -50
+				desc = "FABRICATE_HOOK_RULER_TARGET"
+				scope:target = {
+					is_ruler = yes
+					scope:target.top_liege = scope:owner.top_liege
+				}
+			}
+		}
+		
+		# At War
+		first_valid = {
+			modifier = {
+				add = -200
+				desc = "SCHEME_AT_WAR"
+				scope:target = {
+					is_at_war_with = scope:owner
+				}
+			}
+			modifier = {
+				add = -150
+				scope:target = {
+					is_landed = no
+					exists = liege
+					liege = {
+						is_at_war_with = scope:owner
+					}
+				}
+				desc = "SCHEME_AT_WAR_WITH_LIEGE"
+			}
+			modifier = {
+				add = -100
+				scope:target = {
+					is_landed = yes
+					exists = liege
+					liege = {
+						is_at_war_with = scope:owner
+					}
+				}
+				desc = "SCHEME_AT_WAR_WITH_LIEGE"
+			}
+		}
```

Comment:
Reminder these are modifiers to balance abdication. It may be too harsh for this plot.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/116584985-fb031000-a928-11eb-90c2-0acd2afecb8c.png)

</details>